docs/versioned_docs/version-1.x.x/event.md

```diff
@@ -7,7 +7,6 @@ sidebar_label: Event handling
 `react-native-reanimated`'s new syntax is possible to be used with `Animated.event`. Instead of providing only a mapping from event fields to animated nodes, it is allowed to write a function that takes reanimated values map as an input and return a block (or any other reanimated function) that will be then used to handle the event.
 
 This syntax allows for providing some post-processing for the event data that does not fit well as a dependency of other nodes we connect to `Animated.View` component props.
-[See example](https://github.com/software-mansion/react-native-reanimated/blob/master/Example/src/PanRotateAndZoom/index.js)
```

Comment:
Could you change this link to the correct one instead of removing it?

https://github.com/software-mansion/react-native-reanimated/blob/master/Example/reanimated1/PanRotateAndZoom/index.js

Reply:
Sure thing